enforcer-rules/src/test/java/com/google/cloud/tools/dependencies/enforcer/LinkageCheckerRuleTest.java

```diff
@@ -164,9 +164,8 @@ public void testExecute_shouldFailForBadProject() throws RepositoryException {
           "The rule should raise an EnforcerRuleException for artifacts missing dependencies");
     } catch (EnforcerRuleException ex) {
       // pass
-      verify(mockLog).error(ArgumentMatchers.startsWith("Linkage Checker rule found 384 errors."));
-      assertEquals(
-          "Failed while checking class path. See above error report.", ex.getMessage());
+      verify(mockLog).error(ArgumentMatchers.startsWith("Linkage Checker rule found 123 errors."));
```

Comment:
Now the dashboard uniquely count the missing class, regardless of method or field references.